sql/qaqc/qaqc_app.sql

```diff
@@ -28,10 +28,13 @@ SELECT
     HPD_review.invalid_date_statusr,
     HPD_review.invalid_date_statusx,
     HPD_review.incomp_tract_home,
-    HPD_review.dem_nb_overlap 
+    HPD_review.dem_nb_overlap,
+    qaqc_app_units.classa_net_mismatch 
```

Comment:
This query is great,  clear that this is where the HPD review and app-specific work come together